tests/test_snmpuser.rb

```diff
@@ -175,14 +175,16 @@ def test_snmpuser_destroy
     # destroy the user
     destroy_user(snmpuser)
     # check user got removed.
+    sleep(5)
+    node.cache_flush
     refute_show_match(command: 'show run snmp all | no-more',
                       pattern: user_pat(name, group))
     assert_nil(SnmpUser.users[name])
   end
 
   def test_snmpuser_auth_password_equal_invalid_param
-    name = 'testV3PwEqualInvalid'
-    auth_pw = 'test1234567'
+    name = 'testV3PwEqualInvalid2'
+    auth_pw = 'TeSt297534'
```

Comment:
Simple passwords are not accpeted on n7k and n6k and we need to have a upper case, lowercase, number etc. combination.